cms/search/tests.py

```diff
@@ -2,6 +2,8 @@
 from django.test import TestCase
 from wagtail.contrib.search_promotions.models import SearchPromotion
 from wagtail.search.models import Query
+import re
```

Comment:
I'm not sure we use this anywhere

Reply:
Thanks, I swapped out a hideous regex for just using lxml :)